php/php.editor/src/org/netbeans/modules/php/editor/completion/DocRenderer.java

```diff
@@ -242,6 +243,7 @@ private static String getLocation(PhpElement indexedElement) {
         private final PhpElement indexedElement;
         private final List<String> links = new ArrayList<>();
         private final ASTNode node;
+        @NullAllowed
         private PHPDocBlock phpDocBlock;
 
         public PHPDocExtractor(CCDocHtmlFormatter header, PhpElement indexedElement, ASTNode node, PHPDocBlock phpDocBlock) {
```

Comment:
Missing `@NullAllowed` for `phpDocBlock` param, right?

Reply:
Now I see that @junichi11 already requested this change, sorry.

Reply:
Yes, @tmysik is right.

Reply:
@KacerCZ 
Missing `@NullAllowed` yet:
```java
public PHPDocExtractor(CCDocHtmlFormatter header, PhpElement indexedElement, ASTNode node, @NullAllowed PHPDocBlock phpDocBlock) {
```

Reply:
Fixed.

Reply:
Thanks!